WordPress/Classes/ViewRelated/Post/PostSettings/PostSettingsViewModel.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import WordPressData
+
+final class PostSettingsViewModel: ObservableObject {
+    let post: AbstractPost
```

Comment:
Since `PostSettingsViewModel` is an `ObservableObject`, should you `@Publish` this property? Otherwise, I guess there will be no update in `PostSettingsViewModel` instances?

Reply:
That's not the intention as the only observable bits we care about are modeled using `@Published var settings: PostSettings`.

> should you @Publish this property

A `@Published` property on a `let` doesn't really do anything.